mpp-vscode/src/providers/chat-view.ts

```
@@ -0,0 +1,324 @@
+Try: \`@code write a function to sort an array\`
+
+Your message: "${content}"`;
+  }
+
+  private async handleAction(action: string, data: any): Promise<void> {
+    this.log(`Action: ${action}, data: ${JSON.stringify(data)}`);
+
+    switch (action) {
+      case 'insert':
+        // Insert code at cursor
+        const editor = vscode.window.activeTextEditor;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)